src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -217,6 +206,16 @@ func runClientProxy(cfg *config.Config) {
 		exit(err)
 	}
 
+	if *clearProxySettings {
+		// This is a workaround that attempts to fix a Windows-only problem where
+		// Lantern was unable to clean the system's proxy settings before logging
+		// off.
+		//
+		// See: https://github.com/getlantern/lantern/issues/2776
+		doPACOff("http://" + cfg.UIAddr + "/proxy_on.pac")
```

Comment:
Oh I didn't notice the check below. It's already checked in [`Config.ApplyDefaults()`](https://github.com/getlantern/lantern/blob/valencia/src/github.com/getlantern/flashlight/config/config.go#L215), so the below check is extra I think.